packages/hub/src/components/layout/RootLayout/PageFooter.tsx

```diff
@@ -1,16 +1,73 @@
 "use client";
 import { FC } from "react";
+import { FaGithub, FaDiscord, FaRss } from "react-icons/fa";
+import { SiGraphql } from "react-icons/si";
+import Image from "next/image";
+import logoPic from "@/public/logo.png";
+import Link from "next/link";
+import {
+  DISCORD_URL,
+  GITHUB_URL,
+  NEWSLETTER_URL,
+  QURI_DONATE_URL,
+} from "@/lib/common";
+import { aboutRoute, privacyPolicyRoute, termsOfServiceRoute } from "@/routes";
+
+const linkClasses = "items-center flex hover:text-gray-900";
 
 export const PageFooter: FC = () => (
-  <div className="p-8 border-t border-t-slate-200 bg-slate-100">
-    <div className="text-sm text-slate-500">
-      By{" "}
-      <a
-        href="https://quantifieduncertainty.org"
-        className="text-blue-500 hover:underline"
-      >
-        QURI
-      </a>
+  <div className="px-8 py-4 border-t border-t-slate-200 bg-slate-100">
+    <div className="text-sm text-slate-400 flex flex-column justify-between max-w-4xl mx-auto">
```

Comment:
`flex-column` here is incorrect, it's laid out as a row (correct tailwind class is `flex-col`).

I suggest something like `flex-col gap-4 md:flex-row` for mobile-friendly version.
<img width="523" alt="Captura de pantalla 2023-07-21 a la(s) 15 19 38" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/a9ae5b10-95bd-4b6c-bd18-ad96d3af1bc5">

Reply:
> `text-slate-400`
(on `bg-slate-100`)

Contrast here is pretty weak:
<img width="345" alt="Captura de pantalla 2023-07-21 a la(s) 15 26 35" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/f0f670a2-f18e-4a29-b948-0ec9a1c27cb1">

(Note that you and I are both on macbooks with great screens, but other people might have significantly worse experience)